api/dtty.json

```diff
@@ -69,6 +69,41 @@
       "email": "paul@thebeatles.com",
       "password": "beatles",
       "id": 10
+    },
```

Comment:
Fucked this up and shouldn't have committed this. 🤷‍♂ Might go back and revert this.